src/SFA.DAS.EmployerApprenticeshipsService.Application/Queries/GetEmployerAccountByHashedId/GetEmployerAccountByHashedIdValidator.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Threading.Tasks;
+using SFA.DAS.EAS.Application.Validation;
+
+namespace SFA.DAS.EAS.Application.Queries.GetEmployerAccountByHashedId
+{
+    public class GetEmployerAccountByHashedIdValidator : IValidator<GetEmployerAccountByHashedIdQuery>
+    {
+        public ValidationResult Validate(GetEmployerAccountByHashedIdQuery item)
+        {
+            var validationResult = new ValidationResult();
+
+            if (string.IsNullOrEmpty(item.HashedAccountId))
+            {
+                validationResult.AddError(nameof(item.HashedAccountId));
```

Comment:
Worth adding to the validation message that the error is that the hashedAccountId is empty or null as there may be additional validation added later on and it will help debug without looking into the code.